spock-mockable/src/main/groovy/io/github/joke/spockmockable/internal/MockableASTTransformation.groovy

```diff
@@ -18,12 +18,14 @@ import static org.codehaus.groovy.control.CompilePhase.OUTPUT
 class MockableASTTransformation extends AbstractASTTransformation implements CompilationUnitAware {
 
     private static Set<String> detectedClasses = [] as Set<String>
+    private static Set<String> detectedPackages = [] as Set<String>
     private static CompilationUnit compilationUnit
 
     @Override
     void visit(ASTNode[] nodes, SourceUnit source) {
         def annotation = nodes[0] as AnnotationNode
         detectedClasses.addAll(getClassList(annotation, 'value')*.name as List)
+        detectedPackages.addAll(getMemberStringList(annotation, 'packages')?:[])
```

Comment:
needs to be `getMemberList` for groovy 2.5 backwards compatibility

Reply:
Make sense. I fixed it.